server/app/assets/javascripts/admin_applications.ts

```diff
@@ -18,7 +18,7 @@ class AdminApplications {
     this.displayFrame = frame
     this.cards = document.querySelectorAll(
       AdminApplications.CARD_SELECTOR,
-    ) as any as Array<HTMLElement>
+    ) as any as Array<HTMLElement> // eslint-disable-line @typescript-eslint/no-explicit-any
```

Comment:
I think in cases like this we should use `as unknown as Array<HTMLElement>` instead of any. And it will help avoiding eslint error.

Reply:
Same here.  I'm trying to change as little as possible.  A good follow up to this PR would be to go back and remove these disables wherever possible.